packages/vm-config/decentral-itest-orchestration-config.json

```diff
@@ -0,0 +1,130 @@
+{
+    "$comment": "This SwingSet config file (see loadSwingsetConfigFile) is designed to bring up vats to test Orchestration. Comapre with decentral-itest-vaults-config.json.",
+    "bootstrap": "bootstrap",
+    "defaultReapInterval": 1000,
+    "coreProposals": [
+        "@agoric/builders/scripts/vats/init-core.js",
+        "@agoric/builders/scripts/vats/init-network.js",
+        "@agoric/builders/scripts/vats/init-localchain.js",
+        "@agoric/builders/scripts/vats/init-transfer.js",
+        "@agoric/builders/scripts/vats/init-transfer.js",
+        "@agoric/builders/scripts/vats/init-orchestration.js",
+        {
+            "module": "@agoric/builders/scripts/inter-protocol/init-core.js",
+            "entrypoint": "defaultProposalBuilder",
+            "args": [
+                {
+                    "econCommitteeOptions": {
+                        "committeeSize": 3
+                    },
+                    "referencedUi": "bafybeidvpbtlgefi3ptuqzr2fwfyfjqfj6onmye63ij7qkrb4yjxekdh3e",
+                    "minInitialPoolLiquidity": "0"
+                }
+            ]
+        },
+        {
+            "module": "@agoric/builders/scripts/inter-protocol/add-collateral-core.js",
+            "entrypoint": "defaultProposalBuilder",
+            "args": [
+                {
+                    "interestRateValue": 1000,
+                    "interchainAssetOptions": {
+                        "denom": "ibc/toyatom",
```

Comment:
The wallet driver depends on that right now,
https://github.com/Agoric/agoric-sdk/blob/f880c468257b99d9275a7ec0e93b55eb90817890/packages/boot/test/bootstrapTests/walletFactory.ts#L39

This was the status quo so I consider changing it out of scope. Many tests use that so it's worth handling in its own PR or in one that will break because of it.